sunpy/io/special/asdf/resources/manifests/sunpy-1.0.0.yaml

```diff
@@ -44,3 +44,12 @@ tags:
 
   - tag_uri: "tag:sunpy.org:sunpy/coordinates/frames/geocentricearthequatorial-1.0.0"
     schema_uri: "asdf://sunpy.org/sunpy/schemas/geocentricearthequatorial-1.0.0"
+
```

Comment:
@braingram When is good practice to make a new manifest vs adding stuff to an existing one?

Reply:
tldr; I'd suggest adding a new manifest for these changes
Adding a new manifest (and using it to create a new extension `sunpy-1.10`) has the benefit of informing users of old software that a new version (that supports these new features) is available.
This can be seen by generating 2 different files, one with a new manifest and one with an updated manifest (both containing a `SolarMagnetic` frame in this case). The files differ only in the extension metadata in the asdf header. For the updated manifest the file contains
```yaml
#ASDF 1.0.0
#ASDF_STANDARD 1.5.0
%YAML 1.1
%TAG ! tag:stsci.edu:asdf/
--- !core/asdf-1.1.0
asdf_library: !core/software-1.0.0 {author: The ASDF Developers, homepage: 'http://github.com/asdf-format/asdf',
  name: asdf, version: 3.0.1}
history:
  extensions:
  - !core/extension_metadata-1.0.0
    extension_class: asdf.extension._manifest.ManifestExtension
    extension_uri: asdf://asdf-format.org/core/extensions/core-1.5.0
    software: !core/software-1.0.0 {name: asdf, version: 3.0.1}
  - !core/extension_metadata-1.0.0
    extension_class: asdf.extension._manifest.ManifestExtension
    extension_uri: asdf://sunpy.org/extensions/sunpy-1.0.0
    software: !core/software-1.0.0 {name: sunpy, version: 5.1.0}
c: !<tag:sunpy.org:sunpy/coordinates/frames/solarmagnetic-1.0.0>
  frame_attributes: {magnetic_model: igrf13}
...
```
Using a new manifest, the file differs only by the extension_uri for the new extension:
```yaml
    extension_uri: asdf://sunpy.org/extensions/sunpy-1.1.0
```
Opening either of these files does not work with sunpy main (as the converters are not supported by the existing `1.0.0` extension). However, the warnings that are reported differ for the two files. For the updated manifest, asdf warns:
```
AsdfConversionWarning: tag:sunpy.org:sunpy/coordinates/frames/solarmagnetic-1.0.0 is not recognized, converting to raw Python data structure
```
For the new manifest option, the file gives 2 warnings:
```
AsdfConversionWarning: tag:sunpy.org:sunpy/coordinates/frames/solarmagnetic-1.0.0 is not recognized, converting to raw Python data structure
AsdfWarning: File 'new_manifest.asdf' was created with extension URI 'asdf://sunpy.org/extensions/sunpy-1.1.0' (from package sunpy==5.1.0), which is not currently installed
```
Note that the warning only seen for the new manifest indicates that something (in this case the `sunpy-1.1.0` extension) is not installed. Additionally, if the files are opened with `strict_extension_check` enabled only the one created with the new manifest will produce an error.
Let me know if more details would be helpful.